rules/windows/process_creation/proc_creation_win_rar_susp_greedy_compression.yml

```diff
@@ -28,14 +28,14 @@ detection:
             - ' -r ' # recursive
     selection_cli_folders:
         CommandLine|contains:
-            - ' :\\\*.'
-            - ' :\\\\\*.'
-            - ' :\Users\Public\'
+            - ' ?:\\\*.'
+            - ' ?:\\\\\*.'
+            - ' ?:\$Recycle.bin\'
+            - ' ?:\PerfLogs\'
+            - ' ?:\Temp'
+            - ' ?:\Users\Public\'
+            - ' ?:\Windows\'
```

Comment:
Some ideas/pointers:

1. can't we just remove the space and don't need the wildcard. Should be more inclusive, if possible
2. why is there a dot after the `:\*` and `:\\*`?